psh/echo/echo.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Phoenix-RTOS
+ *
+ * echo - display a line of text
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "../psh.h"
+
+
+void psh_echoinfo(void)
```

Comment:
```suggestion
static void psh_echoinfo(void)
```
I guess that since this is registered as callback function, this could be `static`. Unfortunately, most of our applets, don't use a `static` keyword here, since we had to import them before and it just remained this way.

Reply:
It applies to `psh_echo()` as well.